tests/core_tests/test_templatekey.py

```diff
@@ -428,7 +428,7 @@ def test_subset_format(self):
         )
 
         for test in tests:
-            print(test)
+            # print(test)  # Causes UnicodeEncodeError on Windows
```

Comment:
I check the history of the file. This print has been present since the first commit of this function. It's likely to have been forgotten after debug prior to merging.
Instead of commenting the line, I suggest we remove the line.

Reply:
Done!